### PR TITLE
update write log & add func MergeMaps

### PR DESCRIPTION
**Pull request by willy182**

### CODE REVIEW

**Comment by bmd-antoniush** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://build.bhinnekalocal.com:8080//job/Golib-PR/20/

File: helper.go [hunk 1]
```diff
@@ -469,3 +469,21 @@ func CamelToLowerCase(s string) string {
 	}
 	return strings.ToLower(strings.Join(a, " "))
 }
+
+// MergeMaps func to merge map[string]interface{}
+func MergeMaps(map1, map2 map[string]interface{}) map[string]interface{} {
+	result := make(map[string]interface{})
+	for k, v := range map1 {
+		if _, ok := map1[k]; ok {
```
Comment (agungdwiprasetyo) [inline helper.go]: mas gaperlu ngecek exist apa engga soalnya udah pasti exist karena ini nge-foreach diri sendiri (`map1`)

File: helper.go [hunk 2]
```diff
@@ -469,3 +469,21 @@ func CamelToLowerCase(s string) string {
...
+			result[k] = v
+		}
+	}
+
+	for k, v := range map2 {
+		if _, ok := map2[k]; ok {
```
Comment (agungdwiprasetyo) [inline helper.go]: yg ini juga mas

**Comment by bmd-antoniush** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://build.bhinnekalocal.com:8080//job/Golib-PR/21/

**Pull request merged**